alerter_test.py

```diff
@@ -0,0 +1,8 @@
+from alerter import *
+from alerter import alert_failure_count
+
+alert_in_celcius(400.5)
+alert_in_celcius(303.6)
+assert (alert_failure_count == 2)
```

Comment:
How do we know the production code works when the value is within range? it should be ascertained by including a test case.